src/qt/participantdialog.cpp

```diff
@@ -257,3 +263,84 @@ void ParticipantDialog::on_monitorServiceCheckBox_stateChanged(
     m_options->m_monitor_service = arg1;
     mp_sd->setOptions(*m_options);
 }
+
+void ParticipantDialog::on_lossSpin_valueChanged(
+        int arg1)
+{
+    m_options->m_lossPerc = arg1;
+    mp_sd->setOptions(*m_options);
+    std::cout << "perc " << m_options->m_lossPerc << std::endl;
```

Comment:
Remove debugging print
```suggestion
    m_options->m_lossPerc = arg1;
    mp_sd->setOptions(*m_options);
```